JPF/utils.py

```diff
@@ -134,10 +134,11 @@ def hpd_csv2sql(description, input_csv_url, sep_char,\
             return
 
         # stream csv into chunked dataframes
-        log.info("Streaming CSV from {} ...".format(input_csv_url))
+        log.info("Streaming {} CSV fields from {} ...".format(
+            len(keep_cols), input_csv_url))
 
         csv_chunks = pd.read_csv(input_csv_url, sep=sep_char, dtype=dtype_dict,\
-                encoding='utf8', chunksize=csv_chunk_size)
+                encoding='utf8', chunksize=csv_chunk_size, usecols=keep_cols)
```

Comment:
Sweet.

For those following along, here's the docs for [pandas.read_csv](http://pandas.pydata.org/pandas-docs/stable/generated/pandas.read_csv.html)